test/utils/course_utils.spec.js

```diff
@@ -32,6 +32,18 @@ describe('courseUtils.generateTempId', () => {
   });
 });
 
+describe('CourseUtils.courseSlugRegex', () => {
+  it('returns a regex that validates course slug', () => {
+    const courseSlugRegex = courseUtils.courseSlugRegex();
+    expect(courseSlugRegex.test(' valid ')).to.eq(true);
```

Comment:
Using more realistic valid values here would be useful. We're using this to test the 'term', 'school', and 'title' fields from user input, so we should use some of the kinds of real values we typically see, including some non-latin scripts.

You can find plenty of examples here: https://outreachdashboard.wmflabs.org/campaigns/miscellanea/programs